stix2validator/validator.py

```diff
@@ -295,42 +355,39 @@ def validate_file(fn, options=None):
         options: An instance of ``ValidationOptions``.
 
     Returns:
-        An instance of ValidationResults.
+        An instance of FileValidationResults.
 
     """
-    results = ValidationResults(fn=fn)
+    file_results = FileValidationResults(filepath=fn)
     output.info("Performing JSON schema validation on %s" % fn)
 
     if not options:
         options = ValidationOptions(files=fn)
 
     try:
         with open(fn) as instance_file:
-            instance = json.load(instance_file)
+            file_results.object_results = validate(instance_file, options)
 
-        if options.files:
-            results = validate_instance(instance, options)
-
-    except SchemaInvalidError as ex:
-        results.fatal = ValidationErrorResults(ex)
-        msg = ("File '{fn}' was schema-invalid. No further validation "
-               "will be performed.")
-        output.info(msg.format(fn=fn))
     except Exception as ex:
```

Comment:
Error messages like this were wrong.  SchemaInvalidError is documented to represent an error with the schema, not an error with the data.  (And it seemed to be used that way.)  Anyway, the code (now in `validate_parsed_json()`) sets an error message in the `ObjectValidationResults.errors` list, which would get printed out at the end, so I didn't see the need for an extra info message here.  Also, when validating lists of objects, further validation _will_ be performed (on the next object, if there is one), so that warning about "no further validation" seemed wrong too.  So I just dropped the info message altogether.